dev/src/index.ts

```diff
@@ -1176,7 +1174,7 @@ export class Firestore {
     let endCalled = false;
 
     return new Promise((resolve, reject) => {
-      const streamReady = () => {
+      const releaseStream = () => {
```

Comment:
I don't like this name. "released" can be confused with releasing back into the pool. I certainly was confused that way when reading this.

This promise represents that the stream is ready for use or initialized or something like that.